src/components/SearchAndFilter/SearchAndFilter.test.js

```diff
@@ -3,6 +3,25 @@ import React from "react";
 
 import SearchAndFilter from "./SearchAndFilter";
 
+const sampleData = [
+  {
+    heading: "Clouds",
+    chips: [{ value: "Google" }, { value: "AWS" }, { value: "Azure" }],
```

Comment:
missing `lead` in this data set as well.

Reply:
The Chip element itself has tests to cover lead and non-lead so testing that here is probably not needed.